sync.go

```diff
@@ -419,13 +424,6 @@ func (lw *LibWallet) RescanBlocks() error {
 	return nil
 }
 
-func (lw *LibWallet) CancelRescan() {
-	if lw.syncData.cancelRescan != nil {
```

Comment:
We can as well do away with the `lw.syncData.cancelRescan` variable. It's of no further use with this change.